snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -411,6 +412,21 @@
         }
     }
 
+    // Bug 1238092. Got fixed in Fx45. We can drop this when Fx44
+    // usage drops significantly.
+    function keypressIsMine(snippet) {
+        function onKeyPress(e) {
+            e.stopPropagation();
+        }
+        function attachHandler(matches) {
+            for (var i = 0, len = matches.length; i < len; i++) {
+                matches[i].addEventListener('keypress', onKeyPress);
```

Comment:
I don't think that we'll ever go above 3-4 input elements but maybe we can attach it to `.snippet`?